scripts/Interface/reduction_gui/reduction/diffraction/diffraction_adv_setup_script.py

```diff
@@ -82,7 +82,7 @@ class AdvancedSetupScript(BaseScriptElement):
     @property
     def cache_dir(self):
         """Passing all three candidates back as one list"""
-        return [self.cache_dir_scan_save, self.cache_dir_scan_1, self.cache_dir_scan_2]
+        return ",".join(self.cache_dir_scan_save, self.cache_dir_scan_1, self.cache_dir_scan_2)
```

Comment:
Could you add a comment in this function (don't have to be in the docstring, a regular comment would be sufficient) to state that the reason we are using "," here is because the default deliminator in Mantid is ","?